docker-compose.yml

```diff
@@ -58,8 +58,11 @@ services:
       - DJANGO_ES_URLS=http://elasticsearch:9200
       - DATABASE_URL=postgres://buildhub2:buildhub2@db/buildhub2
       - DEVELOPMENT=1
+      - GOOGLE_APPLICATION_CREDENTIALS=/tmp/credentials
```

Comment:
This is a nit pick, but I'd prefer this to be simply unset by default, and have the mount commented out with an explanation of how to enable it. Mounting arbitrary files isn't useful IMO.

Reply:
I prefer the configuration set up this way, since development only requires `GOOGLE_APPLICATION_CREDENTIALS` to be set on the local machine for the rest of the machinery to work as intended. It's no longer mounting an arbitrary file (the comment is misleading since I modified the line in a previous commit). Just to clarify what this particular statement is doing, the docker-compose file is equivalent to the following command:

```
docker run  \
-v $GOOGLE_APPLICATION_CREDENTIALS:/tmp/credentials
-e GOOGLE_APPLICATION_CREDENTIALS=/tmp/credentials
-it buildhub2:latest
```

The environment variable `GOOGLE_APPLICATION_CREDENTIALS` points to the JSON keyfile on the local disk. This is mounted into the docker container if it exists. In the container, the application will look for a valid keyfile at the location `/tmp/credentials`.

Reply:
Ah actually, I get the following error when unsetting `GOOGLE_APPLICATION_CREDENTIALS`, so maybe I'll add this to the README. `ERROR: Cannot create container for service web: create .: volume name is too short, names should be at least two alphanumeric characters`